configure.ac

```diff
@@ -87,20 +87,6 @@ else
 AM_CONDITIONAL([USE_PMIX], [false])
 fi
 
-# Debug build
-AC_ARG_ENABLE([debug],
-              AC_HELP_STRING([--enable-debug],
-                             [Enable debug build options.]))
```

Comment:
I'm not clear on why the removal of this option. Currently updating our spackage which has a variant for this.

Reply:
It wasn't doing anything useful - see note in the issue description. If someone wants a debug build, they should use `configure <config-opts> "CFLAGS=-g -O0"` or similar

Reply:
The actual CFLAGS to pass are compiler dependent.